Bottom: 0e96472563e880dd921493cb5b89fc52ddf2c9e8
Top:    d3340396808a0f2e1d9083cd3d102c12c95873e9
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-27 12:24:38 -0400

Refresh of of-soc-simple-rework

---

diff --git a/sound/soc/codecs/tlv320aic26.c b/sound/soc/codecs/tlv320aic26.c
index 3387d9e..95a7ac9 100644
--- a/sound/soc/codecs/tlv320aic26.c
+++ b/sound/soc/codecs/tlv320aic26.c
@@ -487,7 +487,7 @@ static int aic26_spi_probe(struct spi_device *spi)
 
 #if defined(CONFIG_SND_SOC_OF_SIMPLE)
 	/* Tell the of_soc helper about this codec */
-	of_snd_soc_register_codec(&aic26_soc_codec_dev, aic26, &aic26_dai,
+	of_snd_soc_register_codec(&aic26_soc_codec_dev, aic26, &aic26_dai, 1,
 				  spi->dev.archdata.of_node);
 #endif
